onnxruntime/core/mlas/lib/pooling.cpp

```diff
@@ -1196,6 +1196,8 @@ Return Value:
     bool AllPaddingIsZero = true;
     bool AllKernelsAreSmall = true;
 
+    ORT_ENFORCE(Dimensions <= 3, "Bad mlas work block dimensions");
```

Comment:
MLAS can't have a dependency on ORT_ENFORCE: it needs to be able to build standalone.

Reply:
Done.